github_connector/models/github_repository_branch.py

```diff
@@ -109,8 +109,8 @@ class GithubRepository(models.Model):
     )
 
     # Init Section
-    def __init__(self, pool, cr):
-        source_path = self._get_source_path()
+    def __new__(cls, *args, **kwargs):
```

Comment:
This change is not correct. You should use ORM methods. Check if the needed one is `_auto_init`.